### DISCUSSION TITLE
How to zoom in/zoom out the pages

### DISCUSSION THREAD

**Discussion by Vikash-JS** [readium/kotlin-toolkit#474] [Q&A]
Hello, all. Really loved the readium communtiy. Thanks for all the support and for this amazing project.

I need to add a zoom in functionality in our project which is based on education and since we have images in the epub we want to have zoom in functionality as it will be shown in our classrooms on screen panels. Can any one help?

**Comment by qnga** [comment]
Hi,
I guess you're talking about Fixed layout EPUBs. The `EpubNavigator` supports pinch-to-zoom for fixed layout EPUBs, are you interested in something different?

**Reply by mickael-menu** [reply]
It could be also the "pinch/tap to zoom images" in a reflowable EPUBs, which is not supported in the Readium toolkits at the moment.

For this, I thought of having additional information in the input event contexts, such as a representation of the `<img>` element and its frame in the view coordinates. From there, an app can use `publication.get(href)` to get the content of the image and handle the zooming action itself.

**Reply by Vikash-JS** [reply]
Well thanks anyway
